gui/styles/_markdown_viewer.styl

```diff
@@ -0,0 +1,110 @@
+black = #000
+chablis = #fff2f2
+charcoalGrey = #32363f
+coolGrey = #95999d
+paleGrey = #f5f6f7
+pomegranate = #f52d2d
```

Comment:
Come on! Our only source of colors should be cozy-ui! (https://github.com/cozy/cozy-ui/blob/8c26e8a02220bd3816e03893441429e54d01299e/stylus/settings/palette.styl#L37) 

We should work on using it in the desktop app, right? ;)

Reply:
Totally agreed. But I didn't want to go through the upgrade of `cozy-ui` right now. I'll give it a try and see if it requires work or not :).

Reply:
We tried to upgrade `cozy-ui` but it's too much work for now. We'll continue in https://github.com/cozy-labs/cozy-desktop/pull/1810